packages/fxa-settings/src/components/Settings/FlowRecoveryKeyInfo/en.ftl

```diff
@@ -0,0 +1,14 @@
+## First view in the Recovery-Key-Add UX redesign flow
+
+# The visible title of the container/modal which contains the Recovery Key Add flow
+flow-recovery-key-info-page-title = Account Recovery Key
+# This will be the title of the back button so that users know where that button will take them
+flow-recovery-key-info-back-button-title = Settings
+# The header of the first view in the Recovery Key Add flow
+flow-recovery-key-info-header = Create an account recovery key in case you forget your password
+# In the first view of the Recovery Key Add flow, this is the first of two bullet points explaining why the user should create a recovery key
+flow-recovery-key-info-shield-bullet-point = We encrypt browsing data –– passwords, bookmarks, and more. It’s great for privacy, but it means we can’t recover your data if you forget your password.
+# In the first view of the Recovery Key Add flow, this is the second of two bullet points explaining why the user should create a recovery key
+flow-recovery-key-info-key-bullet-point = That’s why creating an account recovery key is so important –– you can use your key to get your data back
```

Comment:
I can't find a usage for `flow-recovery-key-info-key-bullet-point` (Ctrl+C => Ctrl+F => Ctrl+V). But I did find this match on the string value w/ a different `id`:

```jsx
          <KeyIconListItem>
            <FtlMsg id="recovery-key-add-wizard-flow-first-step-key-bullet-point">
              <p className="text-sm">
                That’s why creating an account recovery key is so important ––
                you can use your key to get your data back
              </p>
            </FtlMsg>
          </KeyIconListItem>
```